x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportOpenJobAction.java

```diff
@@ -611,7 +607,8 @@ public ClusterState execute(ClusterState currentState) {
 
             @Override
             public void onFailure(String source, Exception e) {
-                listener.onFailure(e);
+                logger.error(source + "Failed to clear finished_time due to [" + e.getMessage() + "]", e);
```

Comment:
We tend to start the messages with the job_id. So, this could become:

`logger.error("[" + jobId + "] Failed to clear finished_time; source [" + source + "]", e);`